src/utils.tsx

```diff
@@ -12,9 +12,9 @@ import { getCodeLineAnchor } from './components/CodeView/utils';
 import tracking from './tracking';
 
 // This is how many lines of code it takes to slow down the UI.
-export const SLOW_LOADING_LINE_COUNT = 2000;
-// This is how many characters we will include in trimmed minfied file content.
-export const MINIFIED_FILE_TRIMMED_CHAR_COUNT = 2000;
+export const SLOW_LOADING_LINE_COUNT = 5000;
+// This is how many characters we will include in trimmed content.
+export const TRIMMED_CHAR_COUNT = 300;
```

Comment:
I'm sure you discussed this in your meeting, but why the big discrepancy between the `SLOW_LOADING_LINE_COUNT` of 5000 and `TRIMMED_CHAR_COUNT` - you have a file that's 5000 lines long, with, say 100,000  characters that's displayed, but if there's an extra line return so it's 5001 lines then we only display 300 characters?

Reply:
Truncated content does not give any useful information to reviewers and it is actually dangerous because if we give too much to reviewers, they might not open the entire file. That's why we have this big discrepancy. By having a small number of characters, we can display the truncated view very fast.

5k lines is what we consider large files that are slow to render in code-manager by default.